src/main/java/ru/skypro/lessons/springboot/weblibrary/service/EmployeeService.java

```diff
@@ -1,13 +1,24 @@
 package ru.skypro.lessons.springboot.weblibrary.service;
 
-import ru.skypro.lessons.springboot.weblibrary.pojo.Employee;
+import ru.skypro.lessons.springboot.weblibrary.dto.EmployeeDTO;
+import ru.skypro.lessons.springboot.weblibrary.dto.EmployeeFullInfo;
 
 import java.util.List;
+import java.util.Optional;
 
 public interface EmployeeService {
-    List <Employee> getAllEmployees();
+    List <EmployeeFullInfo> getAllEmployees();
     Integer findSumSalaries();
-    Employee findMinSalary();
-    Employee findMaxSalary();
-    List <Employee> findHighSalary();
+    Optional <EmployeeFullInfo> findMinSalary();
+    Optional<EmployeeFullInfo> findMaxSalary();
+    List <EmployeeFullInfo> findAboveAverageSalary();
+    void addEmployees(EmployeeDTO employeeDTO);
+    void editEmployee(EmployeeDTO employeeDTO);
+    EmployeeDTO getEmployeeById(Integer id);
+    void deleteEmployeeById(Integer id);
+    List <EmployeeFullInfo> getEmployeeWithSalaryHigherThan(Integer salary);
+    List<EmployeeFullInfo> getEmployeeByPosition(Integer position);
+    EmployeeFullInfo getEmployeeByIdFullInfo(Integer id);
+    List<EmployeeDTO> getEmployeeByPage(Integer page);
+    EmployeeDTO findTheHighestSalary();
```

Comment:
Соответственно, тут тоже будет список. 
```suggestion
    List<EmployeeDTO> findTheHighestSalary();
```